motor/docstrings.py

```diff
@@ -1186,3 +1182,79 @@ async def coro():
 
 .. versionadded:: 1.2
 """
+
+unlock_doc = """**DEPRECATED**: Unlock a previously locked server.
+
+:Parameters:
+  - `session` (optional): a
+    :class:`~motor.motor_tornado.MotorClientSession`.
+
+Deprecated. Users of MongoDB version 3.2 or newer can run the
+`fsyncUnlock command`_ directly with
+:meth:`~motor.motor_tornado.MotorDatabase.command`::
+
+     await motor_client.admin.command('fsyncUnlock')
+
+Users of MongoDB version 3.0 can query the "unlock" virtual
+collection::
+
+    await motor_client.admin["$cmd.sys.unlock"].find_one()
+
+.. versionchanged:: 2.2
+   Deprecated.
+
+.. _fsyncUnlock command: https://docs.mongodb.com/manual/reference/command/fsyncUnlock/
+"""
+
+reindex_doc = """**DEPRECATED**: Rebuild all indexes on this collection.
+
+Deprecated. Use :meth:`~motor.motor_tornado.MotorDatabase.command`
```

Comment:
This reminds me of the problem noticed in https://jira.mongodb.org/browse/MOTOR-560. Any bright ideas for how to make this point to `motor.motor_tornado.MotorDatabase.command` OR `motor.motor_asyncio.AsyncIOMotorDatabase.command` depending on which API is being viewed?

Reply:
I looked into this and wasn't able to figure a way out. I think our best bet to resolve this is in 3.0 when we move to a framework-agnostic API.